### DISCUSSION TITLE
ESP32 C3 Super Mini подходит?

### DISCUSSION THREAD

**Discussion by AngryPin** [vvip-68/LedPanelWiFi#91] [Вопросы и ответы]
Доброго времени суток! Есть ли возможность запустить проект на ESP32 C3 Super Mini. На Wemos D1 mini. Запустил без проблем, а вот на сабже не хочет не в какую. И маленький оффтом, так какая версию FastLed использовать нужно? В инструкции по тексты 3.6.0, а на скриншоте 3.5.0. Кстати на ESP8226 прикручена 3.6.0 и все работает.

**Comment by vvip-68** [comment]
Про С3 - не знаю, не пробовал. Теоретически - тоже микроконтроллер со своими пинами вывода, Возможно есть какие-то нюансы. На практике - вот вы нам и расскажете - получилось или нет.

На форуме FastLED на гитхабе вроде есть issues что какие-то проблемы совместимости. Поизучайте.
https://community.openmqttgateway.com/t/fastled-seems-broken-for-esp32c3-boards/2278
https://github.com/FastLED/FastLED/issues/1349
Вообще не понимаю, зачем имея кучу проверенных решений люди покупают всякую экзотику а потом мучаются с непониманием что с этим делать...

По картинкам - вы серьезно думаете, что я с выходом каждой новой версии библиотек ринусь обновлять все скриншоты? Вот заняться мне больше нечем. Актуальная информация - в скетче и текстовых инструкциях, если не забуду обновить. В скетче - точно самая актуальная.
